web/src/state_data.ts

```diff
@@ -142,14 +142,6 @@ export const realm_schema = z.object({
         }),
     ),
     realm_edit_topic_policy: z.number(),
-    realm_playgrounds: z.array(
```

Comment:
Misplaced in [#29615(commit)](https://github.com/zulip/zulip/pull/29615/commits/6ed0dc256bd475567eddf2543675b37ded7c4540)